src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup.mdx

```diff
@@ -0,0 +1,235 @@
+
+<Callout variant="tip">
+If you send 100% of your OpenTelemetry data to our Trace API, we store 100% of that data, unless you have a specific rate limit for your organization, or if you send enough data to trigger our default rate limit.
+</Callout>
+
+## Adjust sampling to see more UI data [#adjust-sampling]
+
+If you're using an APM agent and are seeing little or no data when you start drilling beyond the initial page of the map, you may need to adjust the span reservoir to sample more data. The reason for this is that transaction-level data is populated by sampled trace data. For OpenTelemetry, both the initial page and all drill-down pages are populated by sampled data, so you may need to make some adjustments in sampling to get the data you need.
+
+<Callout variant="important">
+Changing the sampling could result is some additional charges. To get an idea of how you might be affected, see how to estimate [ingest costs](/docs/data-apis/manage-data/calculate-data-ingest).
+</Callout>
+
+Here is some information about the different routines for adjusting the reservoir for APM agents, as well as tips for adjusting sampling for OpenTelemetry.
+
+If you need additional help with adjusting sampling, please see these [support options](https://docs.newrelic.com/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings/).
+
+### APM agent sampling [#sampling-agents]
+
+<Callout variant="tip">
+These steps don't apply to classic external services.
+</Callout>
+
+All APM agents have a reservoir that stores spans, and most of these agent reservoirs are configurable. The size of this reservoir affects the likelihood that an agent is able to send all the spans that it creates. See [fragmented traces](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/#fragmented-traces) for more details. 
+
+
+Data for this feature is derived from adjacent client and server spans where external calls are made from one service to another. When an agent hits its span reservoir limit, there's a chance it will drop some spans representing these calls.
+
+The default agent configuration value of 2000 defines the maximum number of events the agent collects per minute. If there are more spans than this number, the agent collects a statistical sampling, essentially degrading the amount of data that is used in the external services map.
+
+If you’re not seeing the type of detail you want in the UI, you can increase the size of the reservoir up to 10,000. Review the following to adjust the agent reservoirs:
+
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        APM agent
+      </th>
+      <th>
+        Documentation
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        C SDK
+      </td>    
+      <td>
+       The reservoir is not currently configurable 
```

Comment:
add period

Reply:
@barbnewrelic, did you mean after "The reservoir is not currently configurable"? Since the other items in the table don't have periods, I thought I'd leave those out for consistency.